pkg/kubectl/resource_printer.go

```diff
@@ -414,6 +430,7 @@ var persistentVolumeColumns = []string{"NAME", "CAPACITY", "ACCESSMODES", "STATU
 var persistentVolumeClaimColumns = []string{"NAME", "STATUS", "VOLUME", "CAPACITY", "ACCESSMODES", "AGE"}
 var componentStatusColumns = []string{"NAME", "STATUS", "MESSAGE", "ERROR"}
 var thirdPartyResourceColumns = []string{"NAME", "DESCRIPTION", "VERSION(S)"}
+var thirdPartyResourceDataColumns = []string{"NAME", "LABELS", "DATA"}
```

Comment:
Do you have kind information?  I could see that being useful if I do `kubectl get foo.thirdparty1.com,bar.thirdparty2.com`